CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## ?.?.?
+
+* Add new commands `teams add-member` and `teams remove-member` for managing team members.
```

Comment:
Nitpick: De andre punktene i fila her har ikke pleid å være på imperativsform, kanskje endre litt på den for å matche bedre med dem?